WcaOnRails/app/models/round.rb

```diff
@@ -4,6 +4,10 @@ class Round < ApplicationRecord
   has_one :event, through: :competition_event
   belongs_to :format
 
+  serialize :time_limit, TimeLimit
```

Comment:
@jfly Serialization seems like a good alternative to all those columns (at least it seems better at the moment).